spec/services/manifest_builder/by_resource_type/manifest_builder_ephemera_spec.rb

```diff
@@ -44,9 +44,9 @@
   context "when an ephemera folder has a transliterated title" do
     subject(:manifest_builder) { described_class.new(query_service.find_by(id: folder.id)) }
     let(:folder) { FactoryBot.create_for_repository(:ephemera_folder, title: ["title"], transliterated_title: ["transliterated"]) }
-    it "includes that in the manifest label" do
+    it "the root label displays the original title only because it is single valued" do
       output = manifest_builder.build
-      expect(output["label"]).to eq ["title", "transliterated"]
+      expect(output["label"]).to eq "title"
```

Comment:
What's the impact on viewer display for this change? Anything?

Reply:
No impact. The viewer only displays one title.